view/frontend/templates/page/analytics.phtml

```diff
@@ -2,11 +2,13 @@
 /**
  * @see Metrilo\Analytics\Block\Analytics
  */
-$event = $block->getEvent();
-$url   = $block->getLibraryUrl();
+$event           = $block->getEvent();
+$url             = $block->getLibraryUrl();
+$addToCartEvents = $block->getCatalogSession()->getData('metrilo_add_to_cart', true);
 ?>
 <script type="text/javascript">
     require(['<?php echo $url; ?>'], function(){
+        <?php if($addToCartEvents){ $block->cartEventsLoop($addToCartEvents, 'addToCart'); } ?>
```

Comment:
Why do we need this different behaviour?

Reply:
There is no system action (per request) for addTo/removeFrom cart, because it's done via ajax, so an observer is used instead. The observer itself is pushing the data to a session.

Reply:
ok, fair point.

I think it is weird to call the loop like that. Can't you just iterate over the events in the sessions and just `echo` them?